packages/embark-plugin-cmd/package.json

```
@@ -0,0 +1,58 @@
+{
+  "name": "embark-plugin-cmd",
+  "version": "4.1.0-beta.0",
+  "author": "Iuri Matias <iuri.matias@gmail.com>",
+  "contributors": [],
+  "description": "Plugin installation utility for Embark",
+  "homepage": "https://github.com/embark-framework/embark/tree/master/packages/embark-plugin-cmd#readme",
+  "bugs": "https://github.com/embark-framework/embark/issues",
+  "license": "MIT",
+  "repository": {
+    "directory": "packages/embark-plugin-cmd",
+    "type": "git",
+    "url": "https://github.com/embark-framework/embark.git"
+  },
+  "files": [
+    "dist"
+  ],
+  "main": "./dist/index.js",
+  "scripts": {
+    "build": "cross-env BABEL_ENV=node babel src --extensions \".js,.ts\" --out-dir dist --root-mode upward --source-maps",
+    "ci": "npm run qa",
+    "clean": "npm run reset",
+    "lint": "npm-run-all lint:*",
+    "lint:js": "eslint src/",
+    "lint:ts": "tslint -c tslint.json \"src/**/*.ts\"",
```

```suggestion
    "// lint:ts": "tslint -c tslint.json \"src/**/*.ts\"",
```

Need to disable `"lint:ts":` script for now.